src/Hamburger/Hamburger.scss

```diff
@@ -0,0 +1,81 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.menu-item {
+  font-size: 20px;
+  color: $text-color;
+  margin-bottom: 5px;
+}
+
+.burger-icon {
+  margin-right: 15px;
+}
+
+/* Position and sizing of burger button */
+.bm-burger-button {
+  position: fixed;
```

Comment:
Is position fixed the piece that is making the hamburger scroll down with the page?